### PR TITLE
Added BungeeManager.java

### PR DESCRIPTION
**Pull request by IoyoCode**
Added a new BungeeManager.java class that helps you manage your proxy
Planning on adding new features to this soon

### CODE REVIEW

**Comment by joeecodes** [comment]
Please make the target branch 2.0

File: src/main/java/dev/negativekb/api/proxyutils/BungeeManager.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    }
+    /**
+     * Connect a player to a server!
+     *
+     * @param player The player that you would like to connect.
+     * @param server The server that you would like the Player to be connected to.
+     *
+     * @throws IOException if {@code server} is {@code null}
+     */
+    public void connect(Player player, String server) throws IOException {
```
Comment (NegativeKB) [inline src/main/java/dev/negativekb/api/proxyutils/BungeeManager.java:28]: Please provide annotations to declare parameters in the method as Nullable or NotNull.

For this I would annotate both as @NotNull

Example:
public voic connect(@NotNull Player player, @NotNull String server)

**Review by NegativeKB — CHANGES_REQUESTED** [review]

**Comment by NegativeKB** [comment]
I'm also not able to accept this merge unless you would want to merge with our 2.0.0 branch. The branch has a refactor with the packaging so you may need to rewrite this class again.

File: src/main/java/dev/negativekb/api/proxyutils/BungeeManager.java (hunk 1 above)
Comment (IoyoCode) [reply src/main/java/dev/negativekb/api/proxyutils/BungeeManager.java:28]: Should be fixed now.

**Review by NegativeKB — APPROVED** [review]

**Comment by IoyoCode** [comment]
i think i've fixed the conflicts as well now

**Pull request merged**